packages/protocol/index.js

```diff
@@ -1,20 +1,29 @@
 const AirnodeArtifact = require('./artifacts/contracts/Airnode.sol/Airnode.json');
 const ConvenienceArtifact = require('./artifacts/contracts/Convenience.sol/Convenience.json');
-
-const networkChainIds = { ropsten: 3, rinkeby: 4 };
+const MockAirnodeClientArtifact = require('./artifacts/contracts/mock/MockAirnodeClient.sol/MockAirnodeClient.json');
 
 const AirnodeAddresses = {};
 const ConvenienceAddresses = {};
-for (const network in networkChainIds) {
-  const AirnodeDeployment = require(`./deployments/${network}/Airnode.json`);
-  AirnodeAddresses[networkChainIds[network]] = AirnodeDeployment.receipt.contractAddress;
-  const ConvenienceDeployment = require(`./deployments/${network}/Convenience.json`);
-  ConvenienceAddresses[networkChainIds[network]] = ConvenienceDeployment.receipt.contractAddress;
-}
+let AirnodeDeployment, ConvenienceDeployment;
+// Ropsten - 3
+AirnodeDeployment = require(`./deployments/ropsten/Airnode.json`);
```

Comment:
No need for backticks here anymore (and for the other `require`s)
```suggestion
AirnodeDeployment = require('./deployments/ropsten/Airnode.json');
```